book/src/redundancy.md

```diff
@@ -0,0 +1,106 @@
+# Redundancy
+
+[subscribe-api]: https://ethereum.github.io/eth2.0-APIs/#/Validator/prepareBeaconCommitteeSubnet
+
+There are three places in Lighthouse where redundancy is notable:
+
+1. ✅ GOOD: Using a redundant Beacon node in `lighthouse bn --beacon-nodes`
+1. ✅ GOOD: Using a redundant Eth1 node in `lighthouse bn --eth1-endpoints`
+1. ☠️ BAD: Running redundant `lighthouse vc` instances with overlapping keypairs.
+
+I mention (3) since it is unsafe and should not be confused with the other two
+uses of redundancy. **Running the same validator keypair in more than one
+validator client (Lighthouse, or otherwise) will eventually lead to slashing.**
+See [Slashing Protection](./slashing-protection.md) for more information.
+
+From this paragraph, this document will *only* refer to the first two items (1, 2). We
+*never* recommend that users implement redundancy for validator keypairs.
+
+## Redundant Beacon Nodes
+
+The `lighthouse bn --beacon-nodes` flag allows one or more comma-separated values:
+
+1. `lighthouse vc --beacon-nodes http://localhost:5052`
+1. `lighthouse vc --beacon-nodes http://localhost:5052,http://192.168.1.1:5052`
+
+In the first example, the validator client will attempt to contact
+`http://localhost:5052` to perform duties. If that node is not contactable, not
+synced or unable to serve the request then the validator client may fail to
+perform some duty (e.g., produce a block or attest).
+
+However, in the second example, any failure on `http://localhost:5052` will be
+followed by a second attempt using `http://192.168.1.1:5052`. This
+achieves *redundancy*, allowing the validator client to continue to perform its
+duties as long as *at least one* of the beacon nodes is available.
+
+There are a few interesting properties about the list of `--beacon-nodes`:
+
+- *Ordering matters*: the validator client prefers a beacon node that is
+	earlier in the list.
+- *Synced is preferred*: the validator client prefers a synced beacon node over
+	one that is still syncing.
+- *Failure is sticky*: if a beacon node fails, it will be flagged as offline
```

Comment:
Does it retry the first beacon node after the current slot ends?

Reply:
Yep, it does! I re-worded the paragraph to express this :)